runtimes/dotnet-6.0/Example/Index.cs

```diff
@@ -1,24 +0,0 @@
-using Newtonsoft.Json;
```

Comment:
This should be updated instead of removed

Reply:
It was moved to `example` folder (lower-case) for consistency with other languages.